sdk/include/psdk/shobjidl.idl

```diff
@@ -225,6 +225,9 @@ interface IShellFolder : IUnknown
     cpp_quote("#define STR_PARSE_TRANSLATE_ALIASES L\"Parse Translate Aliases\"")
     cpp_quote("#define STR_DONT_PARSE_RELATIVE L\"Don't Parse Relative\"")
 
+    cpp_quote("#define SHCIDS_COLUMNMASK 0x0000FFFFL")
+    cpp_quote("#define SHCIDS_BITMASK 0xFFFF0000L")
```

Comment:
Are these present in the official psdk shobjidl.idl file?

Reply:
In the SDK for Windows 10 they are in ShObjIdl_core.idl (I don't remember when they added that _core file).

Reply:
> For "consistency" purposes it may be of interest to move these

They belong where IShellFolder2 is defined IMHO.

Reply:
Two other missings are added.
Moved.